content/influxdb/v2.0/reference/cli/influx/write/_index.md

```diff
@@ -2,7 +2,7 @@
 title: influx write
```

Comment:
As a best practice globally, we may want to start including overview links at top, so if folks land on this page, they immediately know examples exist.

Reply:
Agree. There's actually a way to have Hugo do this. I'm going to leave this as is for now and incorporate that work in another PR.